modules/kafka/src/main/scala/higherkindness/mu/format/Decoder.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2017-2020 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package higherkindness.mu.format
```

Comment:
Maybe `Encoder` and `Decoder` should be moved somewhere under the `higherkindness.mu.kafka` package? Even though they are already namespaced by being in the `kafka` module, just looking at the FQN `higherkindness.mu.format.Decoder` doesn't give you any hint that it's Kafka-related.

Reply:
Well, I don't think they need to be related. I put `h.m.format` package inside `kafka` project for now but this package only contains serialisation related code. In fact as pointed by @fedefernandez, the serialisation code is duplicated else where in `mu-scala` and we just keep one copy and put somewhere more appropriate. mu-format? mu-marshaller?

Reply:
Can we use the `internal` module? If not, I'm ok with `mu-format` but I'd avoid extra modules as possible.

Reply:
@fedefernandez true. I am happy to move it to `internal` and remove the duplicate code. We can always break it out to its own module as & when necessary.